scout/load/panel.py

```diff
@@ -117,27 +118,33 @@ def _panelapp_panel_ids() -> List[str]:
     return [panel_info["Panel_Id"] for panel_info in json_lines.get("result", [])]
 
 
-def _parse_panelapp_panel(adapter, panel_id, institute, confidence):
+def _parse_panelapp_panel(
+    panel_id: str,
+    institute: str,
+    confidence: str,
+    ensembl_id_to_hgnc_id_map: Dict[str, int],
+    hgnc_symbol_to_ensembl_id_map: Dict[str, str],
+):
     """fetch and parse lines from a PanelApp panel, given its ID
 
     Args:
         adapter(scout.adapter.MongoAdapter)
         panel_id(str): The panel app panel id
         confidence(str enum green|amber|red): traffic light-style PanelApp level of confidence
+        ensembl_to_hgnc_map: dict[str, int]
+        hgnc_to_ensembl_map: [int, str]
+
 
     Returns:
         parsed_panel(dict). Example:
             {'version': 3.3, 'date': datetime.datetime(2023, 1, 31, 16, 43, 37, 521719), 'display_name': 'Diabetes - neonatal onset - [GREEN]', 'institute': 'cust000', 'panel_type': 'clinical', 'genes': [list of genes], 'panel_id': '55a9041e22c1fc6711b0c6c0'}
 
     """
-    ensembl_gene_hgnc_id_map: Dict[str, int] = adapter.ensembl_to_hgnc_id_mapping()
```

Comment:
This was really bad, it was fetching both mappings for each PanelApp panel that was parsed :flushed: